### DISCUSSION TITLE
Support Create?

### DISCUSSION THREAD

**Discussion by Cha0sPudd1ng** [shedaniel/RoughlyEnoughItems#1319] [Q & A] [answered]
Hello,
does in the future REI support Create?
JEI supports Create but I find REI better.^^

**Comment by Cha0sPudd1ng** [comment, marked as answer]
Just discovered that REI supports Create in the mod loader Fabric.
It's probably a bug in the Forge version.
Since Fabric is the better choice anyway and all my other mods are also available for Fabric, I switched.

**Reply by shedaniel** [reply]
In order to use REI on Forge with mod support, you need to also have the REI Plugin Compatibilities mod

**Reply by Cha0sPudd1ng** [reply]
Thx for your help. :)
But I'm sticking with Fabric. Because some new mods I use are only available for Fabric.

**Reply by Ae-Mc** [reply]
What about neoforge?

**Reply by JuniorWolfgamingDE** [reply]
Still not working on neoforge :c

**Reply by axieum** [reply]
REI Plugin Compatibilities isn't available for 1.21+ anymore. Is there an alternative by any chance @shedaniel for viewing Create recipes in REI?

_REI is unmatched, it's beautiful and I'd love to keep using it. Thanks for creating REI!_

**Reply by Steelhalo** [reply]
Has anyone got it to work on neoforge 1.21? I love rei and don't want to switch to another recipe viewer. The REI Plugin Compatibilities mod won't work for me it will just crash so I am asking if there is another way as create isn't updated to 1.21.

**Reply by axieum** [reply]
@Steelhalo I ended up switching to EMI with TooManyRecipeViewers - it requires a bit of finesse to configure to be on par with REI's UX but it gets the job done for 1.21.1.

**Reply by peaksuperior885** [reply]
hey so i am kinda late but i made a simple compat mod for forge and neoforge soon it's a port of the fabric one lol

https://github.com/peaksuperior885/Rejei/releases/

**Reply by GasparLiboreiro** [reply]
hello peaksuperior885 and thankyou, is it for 1.21.1?
